edc-extensions/dataplane/dataplane-token-refresh/token-refresh-core/src/main/java/org/eclipse/tractusx/edc/dataplane/tokenrefresh/core/DataPlaneTokenRefreshServiceImpl.java

```diff
@@ -270,6 +270,11 @@ public Result<Void> revoke(String transferProcessId, String reason) {
     }
 
     private Result<Void> deleteTokenData(AccessTokenData tokenData) {
+        var deletionResult = vault.deleteSecret(tokenData.id());
+        if (deletionResult.failed()) {
+            return Result.failure(deletionResult.getFailureMessages());
```

Comment:
no need to instantiate a new `Result`:
```suggestion
            return deletionResult;
```